public/v4/apps/shiori.yml

```diff
@@ -0,0 +1,31 @@
+captainVersion: 4
+
+services:
+  $$cap_appname:
+    image: radhifadlillah/shiori@sha256:3d1eb1f0aeb9804dea41f3ec5d29de5cf214bb66ef04dce1aa942ca17caec7bb
+    environment:
+      SHIORI_DIR: /data
+    volumes:
+      - $$cap_appname-shiori-data:/data
+    restart: on-failure
+    caproverExtra:
+      containerHttpPort: '8080'
+
+caproverOneClickApp:
+  isOfficial: false
+  description: "A simple bookmark manager built with Go."
+  instructions:
+    start: >
+      This app stores its data in a SQLite database in a Docker volume.
+
+      Once deployed, this app will be accessible with default credentials. If
```

Comment:
Let's call out what the default creds are:
```
username: shiori
password: gopher
```

Reply:
I've updated the instructions so that the username and password are in both the "start" and "end" pages.

Thanks for the feedback 🙂